### ISSUE TITLE
Add 8.9.1 cardano node image to armadaalliance docker hub

### ISSUE THREAD

**Issue by TheStophe** [armada-alliance/cardano-node-binaries#5]
Update image / docker pull for 8.9.1

https://hub.docker.com/r/armadaalliance/armada-cn/tags

**Comment by AstroWa3l** [comment]
Would you like me to change the main guides repo? Just wondering if that is all I need to do or if there are more things you want updated.

**Comment by ccgarant** [comment]
Sorry that was written pretty quickly, and I'm not sure who owns the docker hub. 

I just noticed the latest tag in the docker hub was 8.7.2, and I'm not sure who adds the latest and greatest images, but to add 8.9.1 image to docker hub.

https://hub.docker.com/r/armadaalliance/armada-cn/tags

**Comment by AstroWa3l** [comment]
Yeah, that docker image is provided by @jterrier84. I will update the guides though to match his latest release. Thanks for letting us know 🙏

**Comment by ccgarant** [comment]
Oh sorry, I should have probably just mentioned it in the telegram. Good to know.

**Comment by AstroWa3l** [comment]
Don't worry, bruv!
